docs/INSTALLATION PROCESS.md

```diff
@@ -0,0 +1,32 @@
+
+When it comes to setting up a machine with a GPU for deep learning workloads, the first steps primarily involve installing the necessary software components and dependencies. Here's a simplified version of the process:
+
+## Setting Up a Machine with GPU
+
+### 1.Install Operating System:
+Choose a suitable operating system. Ubuntu or other Linux distributions are preferred for deep learning due to their compatibility and control. We will focus on Ubuntu and Debian for this project.
+### 2. Install GPU Drivers:
+Download and install the latest GPU drivers compatible with your GPU model. Visit the GPU manufacturer's website (e.g., NVIDIA or AMD) to get the appropriate drivers.
+
+### 3. Create CONDA Environment:
+Utilize package managers like pip  conda to create a controlled environment. This ensures a clean setup for your deep learning project. For this project we are going to use Miniconda. 
+### 3. Install CUDA Toolkit:
+CUDA is essential for GPU acceleration in deep learning frameworks. Install the CUDA Toolkit, encompassing libraries and tools that optimize GPU performance.
+### 4. Install Deep Learning Framework (TensorFlow):
+Select TensorFlow as the deep learning framework for this project. Install it within the previously created conda environment.
+### 6. Install Additional Libraries:
+Install supplementary libraries like NumPy, Pandas, SciPy,Scykit  and Jupyter Notebook. 
+### 7. Verify Installation:
+Validate our setup by running sample models that exercises our GPU and tests the functionality of the TensorFlow installation.
+
+## 1. Install Operating System:
+In this step, we will thoroughly evaluate our documentation within two distinct scenarios:
+
+### a. Virtual Machine Setup:
+
+For the first scenario, we will conduct our documentation testing on virtual machines. Specifically, we will be utilizing Google Cloud Machines, a powerful cloud-based platform that facilitates the creation of customized instances. This includes tailoring various parameters such as the choice of operating system, regional availability, GPU allocation, and more, based on your precise requirements.
+
+By employing virtual machines, we can emulate diverse computing environments and ensure that our documentation is comprehensive and adaptable to a range of configurations. This allows us to verify the effectiveness of our installation instructions across various settings and configurations.
+
+b. 
```

Comment:
It seems this line got cut off early. Is this intentional? @yamihe19

Reply:
It is intentional.

Reply:
OK no problem! Resolving this issue.